Gamedata/Bluedog_DB/Parts/Titan/bluedog_LR87_11_Single.cfg

```diff
@@ -37,6 +37,8 @@ PART
 	bulkheadProfiles = size1
 
 	tags = ?sm68 lr87 aj11 titan prometheus ldc herakles single engine 1875 1.875
+	
+	techtag = titan3
```

Comment:
This now needs to unlock with Titan I. The upgrades for the single will unlock the Aj5 and AJ11 versions.

Reply:
@zorg2044 fixed in https://github.com/CobaltWolf/Bluedog-Design-Bureau/pull/665/commits/1020e467c36cfc96d15ebaac756a9e1589e52a58